### PR TITLE
Fix legacy picking label bounds

### PR DESCRIPTION
**Pull request by KoBeWi**
Before
image

After
image

### CODE REVIEW

File: scene/gui/color_picker.cpp [hunk 1]
```diff
@@ -1571,19 +1571,16 @@ void ColorPicker::_pick_button_pressed_legacy() {
...
-		picker_window->add_child(picker_preview);
-
 		picker_preview_label = memnew(Label);
-		picker_preview->set_anchors_preset(Control::PRESET_CENTER_TOP);
+		picker_preview_label->set_anchors_preset(Control::PRESET_CENTER_TOP);
 		picker_preview_label->set_text(ETR("Color Picking active"));
-		picker_preview->add_child(picker_preview_label);
 
-		picker_preview_style_box = (Ref<StyleBoxFlat>)memnew(StyleBoxFlat);
+		picker_preview_style_box = Ref<StyleBoxFlat>(memnew(StyleBoxFlat));
```
Comment (AThousandShips) [inline scene/gui/color_picker.cpp:1578]:
```suggestion
		picker_preview_style_box.instantiate();
```

File: scene/gui/color_picker.cpp [hunk 2]
```diff
@@ -1571,19 +1571,16 @@ void ColorPicker::_pick_button_pressed_legacy() {
...
 		picker_preview_style_box->set_bg_color(Color(1.0, 1.0, 1.0));
-		picker_preview->add_theme_style_override(SceneStringName(panel), picker_preview_style_box);
+		picker_preview_style_box->set_content_margin_all(4.0);
+		picker_preview_label->add_theme_style_override("normal", picker_preview_style_box);
```
Comment (AThousandShips) [inline scene/gui/color_picker.cpp:1581]:
```suggestion
		picker_preview_label->add_theme_style_override(CoreStringName(normal), picker_preview_style_box);
```

**Review by bruvzg — APPROVED** [review]
Seems to fix the issue.

Label is off center, and unnecessary huge, but it was like this before.

**Review by AThousandShips — APPROVED** [review]

**Comment by clayjohn** [comment]
Thank you!

**Pull request merged**